tests/model_test.py

```diff
@@ -0,0 +1,77 @@
+__author__='lucaspavanelli'
+
+"""
+Copyright 2022 The aiXplain SDK authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+import time
+from aixtend.utils.config import MODELS_RUN_URL, MODEL_API_KEY
+from aixtend.factories.model_factory import ModelFactory
+
+def test_mt1():
```

Comment:
Call all of these unit tests as functional tests. Mock the backend calls for unit tests.